e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -683,6 +683,45 @@ describe("scenarios > dashboard", () => {
       cy.findByText("There was a problem displaying this chart.");
     });
   });
+
```

Comment:
I don't see a test for the [horizontal overflow](https://github.com/metabase/metabase/assets/6830683/abb42e48-fc8f-44cc-be0a-de68dfd4eeb0), but not sure if we should go into such detail.